api-compact.yaml

```diff
@@ -167,7 +242,7 @@ endpoints:
           operationId: "get_enterprise_catalogs"
           responses: *responses
           x-amazon-apigateway-integration:
-            <<: *apigateway_integration
+            <<: *apigateway_integration_id
```

Comment:
@saleem-latif Yep, but it may confuse someone seeing that it's not being used for the new `enterprise_catalogs` endpoint below :\